pkg/controller/endpointslice/endpointslice_controller.go

```diff
@@ -72,9 +72,9 @@ const (
 	// maxSyncBackOff is the max backoff period for syncService calls.
 	maxSyncBackOff = 1000 * time.Second
 
-	// controllerName is a unique value used with LabelManagedBy to indicated
+	// controllerQualifiedName is a unique value used with LabelManagedBy to indicated
 	// the component managing an EndpointSlice.
-	controllerName = "endpointslice-controller.k8s.io"
+	controllerQualifiedName = "endpointslice-controller.k8s.io"
 
```

Comment:
Is this the normal form of controller names? Or is it a special case? @liggitt

Reply:
this wasn't really rationalized with the controller name constants... it was only meant as a label value

most controllers don't set labels with their names